docs/code/IDataViewDesignPrinciples.md

```diff
@@ -462,7 +462,7 @@ interfaces of their respective worlds.
 In both worlds, there is a cursoring interface associated with the core
```

Comment:
>cursoring interface [](start = 27, length = 19)

Should this be rephrased since `DataVieRowCursor` is not an interface?

Reply:
I will use "mechanism" instead of "interface" :)

Reply:
I will use "mechanism" instead of "interface" where appropriate :)

Reply:
I will use "mechanism" instead of "interface" where appropriate :)

---
In reply to: [368282244](https://github.com/dotnet/machinelearning/pull/4673#discussion_r368282244) [](ancestors = 368282244)